README.md

```diff
@@ -15,3 +15,38 @@ Run `cmake` to configure the build environment and then `make` to build
 To install, use `make install`
 
     sudo make install
+
+## Writing contract files
```

Comment:
This should be title case as in `## Writing Contract Files`

Reply:
Sure, no problem.